More DOM Manipulation/About Me.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="/Users/mac/sei/homeworks/hw-w02d03-dom/More DOM Manipulation/style.css">
```

Comment:
its better if you use relative paths for sources, this will make it easier for your programs to find other files.

similar to how you put the path for your script tag

here is should be 

```
<link rel="stylesheet" href="style.css">
```